resources/style.css

```diff
@@ -0,0 +1,530 @@
+}
+
+.arrow {
+	float: left;
+	margin-right: 15px;
+	width: 0; 
+	border-top: 10px solid transparent;
+	border-bottom: 10px solid transparent;
+}
+
+.arrow-right {
+	border-left: 15px solid #000000;
+}
+
+.arrow-left {
+	border-right: 15px solid #000000;
+}
+
+@keyframes flash {
+	0%   { opacity: 1; }
+	100% { opacity: 0; }
+}
+
+.arrow1 {
+	animation: flash 2.5s infinite;
+} 
+
+.arrow2 {
+	animation: flash 2.5s infinite .5s;
+}
+
+.arrow3 {
+	animation: flash 2.5s infinite 1s;
+} 
+
+.arrow4 {
+	animation: flash 2.5s infinite 1.5s;
+}
+
+.arrow5 {
+	animation: flash 2.5s infinite 2s;
+}
+	
+section .container {
+	max-width: 1300px;
+   	margin: 0 auto;
+	font-size: 2rem;
+	padding: 40px;
+}
+
+.email {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-between;
+}	  
+
+footer {
+	background: #0e0606;
+	height: 100px;
+	margin-bottom: 40px;
+	display: flex;
+	justify-content: space-around; 
+	align-items: center;
+	font-size: 1.3rem;
+	color: #fff;
+	font-weight: 600;
+}
+
+.social-media a { 
+	color:#fff;
+	font-size: 2.5rem;
+}
+
+.icon {
+	margin-right: 50px;
+	padding: 9px;
+}
+
+.icon:hover {
+	color: #e6a3a3;
+	outline: 0.2px solid #453e4c;
+}
+
+/* 
+Extra small <576px (small phones)
+Small ≥576px (phones)
+Medium ≥768px (tablets)
+Large ≥992px (laptops)
+Extra large ≥1200px (desktop monitors) */
+
+@media only screen and (max-width: 1250px)  {
+
+	.container-about-me {
+		width: 800px;
+	}
+
+	.about-me-text {
+		font-size: 1.8rem;
+		right:50px;
+	}
+	.headshot {
+		width: 200px;
+		height: 240px;
+	}
+
+	.container-portfolio,
+	.portfolio-heading {
+		width: 900px;
+	}
+}
+
+@media only screen and (max-width: 992px) {
+
+	.container-about-me {
+		width: 720px;
+	}
+
+	.name-top  h1 {
+		font-size: 2rem;
+	  }
+
+	.title {
+		color: #ffffff;
+		font-size: 3rem;
+	}
+
+	.text {
+		font-size: 1.5rem; 
+	 }
+
+	.see-more {
+		padding: 9px 11px;
+		cursor: pointer;
+	 	font-size: 1rem;
+		border-radius: 5px;
+	}
+
+	.content-about-me {
+		justify-content: center;
+	}
+
+	.about-me-text {
+		font-size: 1.8rem;
+		right: 100px;
+	}
+
+	.portfolio-heading {
+		width: 680px;
+	}
+
+	.container-portfolio {
+		width: 720px;
+	}
+
+	.arrow-holder {
+		margin-top: 90px;
+		margin-left: 60px;
+	}
+
+	.arrow {
+		border-top: 5px solid transparent;
+		border-bottom: 5px solid transparent;
+	}
+	
+}
+
+@media only screen and (max-width: 800px)  {
+
+	.wave-top  {
+		top: 230px;
+		position: absolute;
+	}
+	
+}
+
+@media only screen and (max-width: 750px) {
+
+	.navbar-sticky {
+		display: none;
+	}
+
+	.hamburger-button {
+		background-color: #fff;
+		padding: 10px;
+		border-bottom: 1px solid #a19f9f;
+	}
+
+	.hamburger-button div {
+		display: block;
+		background-color: #6e696a;
+	}
+
+	.container-about-me {
+		margin-top: 90px;
+		width: 80%;
+	}
+
+	.content-about-me {
+		display: flex; 
+		flex-direction: column;
+		align-items: center;
+		box-shadow: 13px 9px 15px -1px rgba(63, 62, 62, 0.32);
+	    border: 1px solid #000000;
```

Comment:
theres still an indentation problem here. its probably that some is spaces and some are tabs. remove all the indentation for this selector and re-add it, that will fix it